.github/scripts/github_metrics.py

```diff
@@ -0,0 +1,132 @@
+        if True not in [l.lower() in label_events for l in expected_label_set]:
+            return False
+    return True
+
+
+def get_triaged_time(label_events):
+    if not is_triaged(label_events):
+        return now
+    timestamps = []
+    for expected_label_set in expected_triage_labels:
+        for label in expected_label_set:
+            if label in label_events:
+                timestamps = timestamps + [label_events[label]]
+    if len(timestamps) == 0:
+        return now
+    return max(timestamps)
+
+
+# using an access token
+g = Github(githubToken)
+
+issues_since=datetime.now()-timedelta(days=30)
+
+total_count = 0
+triaged_count = 0
+triaged_under_5_days_count = 0
+expected_total_time_to_triage = timedelta()
+total_time_to_triage = timedelta()
+
+for repo in my_repos:
+    # discover milestone project
+    issues = [i for i in g.get_repo('dapr/' + repo).get_issues(labels=['kind/bug'], since=issues_since) if i.created_at >= issues_since]
+    total_count += len(issues)
+
+    for issue in issues:
+        events = sorted(issue.get_events(), key=lambda e:str(e.created_at))
+        # Dictionary of the latest time 
+        first_label_events = dict()
+        for event in events:
+            if event.event == 'labeled':
+                canonical_label = event.label.name.lower()
+                if canonical_label not in first_label_events:
+                    first_label_events[canonical_label] = event.created_at
+        triaged_time = get_triaged_time(first_label_events)
+        time_to_triage = triaged_time - issue.created_at
+        expected_total_time_to_triage += time_to_triage
+        if is_triaged(first_label_events):
+            total_time_to_triage += time_to_triage
+            triaged_count += 1
+            if time_to_triage.total_seconds() <= FIVE_DAYS.total_seconds():
+                triaged_under_5_days_count += 1
+            print('Issue %s created %s ago and triaged in %s' % (issue.html_url, humanize.naturaldelta(
+                now - issue.created_at), humanize.naturaldelta(time_to_triage)))
+        else:
+            print('Issue %s created %s ago but still not triaged, assuming %s to triage' % (
+                issue.html_url, humanize.naturaldelta(now - issue.created_at), humanize.naturaldelta(time_to_triage)))
+
+average_days_to_triage = (total_time_to_triage / triaged_count).total_seconds() / ONE_DAY.total_seconds()
+expected_average_days_to_triage = (expected_total_time_to_triage / total_count).total_seconds() / ONE_DAY.total_seconds()
+triaged_under_5_days_ratio = triaged_under_5_days_count / total_count
+
+output = json.dumps({
+    'date': now.strftime('%Y-%m-%d'),
+    'last_30days_total_issues': total_count,
+    'last_30days_total_issues_triaged': triaged_count,
+    'last_30days_total_issues_triaged_in_5_days': triaged_under_5_days_count,
+    'last_30days_total_issues_triaged_in_5_days_percentage': triaged_under_5_days_ratio * 100.0,
```

Comment:
I think this should be the metric for us to track. We should target 90%, currently around 22%